test/imported.test.js

```diff
@@ -6,14 +6,15 @@
 // Comment test cases to get CI pass,
 // will recover them when CI config done
 
-// 'use strict';
+'use strict';
+var describe = require('./describe');
 
-// describe('cloudant imported features', function() {
-//   before(function() {
-//     require('./init.js');
-//   });
+describe('cloudant imported features', function() {
+  before(function() {
+    require('./init.js');
+  });
 
-//   require ('loopback-datasource-juggler/test/include.test.js');
-//   require ('loopback-datasource-juggler/test/crud-with-options.test.js');
-//   require ('loopback-datasource-juggler/test/common.batch.js');
-// });
+  require ('loopback-datasource-juggler/test/include.test.js');
+  require ('loopback-datasource-juggler/test/crud-with-options.test.js');
+  require ('loopback-datasource-juggler/test/common.batch.js');
```

Comment:
It's outside of scope of this pull request, but the "imported features" suite should include tests for operation hooks too. See e.g. https://github.com/strongloop/loopback-connector-mysql/blob/18cffb98ab5a90b8e728019de2b9b050fdc8a383/test/persistence-hooks.test.js